test/vm.install

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# derived from test/image-prepare in cockpit-project/cockpit
+
+import argparse
+import glob
+import os
+import shutil
+import subprocess
+import sys
+
+BOTS_DIR = os.path.realpath(f'{__file__}/../../bots')
+sys.path.append(BOTS_DIR)
+
+missing_packages = "cockpit-ws cockpit-bridge cockpit-storaged fedora-logos"
+# Install missing firefox dependencies.
+# Resolving all dependencies with dnf download is possible,
+# but it packs to many packages to updates.img
+missing_packages_incl_deps = "firefox"
+
+from machine.machine_core import machine_virtual  # NOQA: imported through parent.py
```

Comment:
FTR, there is no `parent.py` any more.

Reply:
Thanks for heads up! Will clean it up in followup